docs/b2500.md

```
@@ -1,390 +1,789 @@
 
-*Note: It will be saved to flash*
+Triggers a full runtime info response.
 
-## 5. Start battery output threshold
+---
 
-### 5.1 Public
+#### Charging mode (V2)
 
-**Topic:**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)